grakn-test/src/test/java/ai/grakn/generator/ConceptIds.java

```diff
@@ -22,7 +22,7 @@
 import ai.grakn.concept.ConceptId;
 
 /**
- * Generator that generates totally concept IDs
+ * Generator that generates totally random concept IDs
  */
 public class ConceptIds extends AbstractGenerator<ConceptId> {
```

Comment:
I agree that it's not ideal but at least with this approach we not forcing *all* the generator tests into the same module. The problem is the Graph API generator tests being in `grakn-test`. We need to resolve that.